src/juicy-popover.html

```diff
@@ -32,270 +32,261 @@
     <div id="container">
         <div id="handle">
             <slot name="handle"></slot>
-            <content select="[slot='handle']"></content>
         </div>
         <div id="expandable">
-            <slot id="deprecated-v1" name="expendable"></slot>
-            <content id="deprecated-v0" select="[slot='expendable']"></content>
             <slot name="expandable"></slot>
-            <content select="[slot='expandable']"></content>
         </div>
     </div>
 </template>
 
 <script>
-    (function (global) {
-        var importDoc;
-        if (document._currentScript) {
-            //@see http://www.polymer-project.org/platform/html-imports.html
-            //@see https://groups.google.com/d/topic/polymer-dev/4UKty9tb1-s/discussion
-            importDoc = document._currentScript.ownerDocument;
-        }
-        else {
-            importDoc = document.currentScript.ownerDocument;
-        }
+    (function (global, document) {
 
-        var template = importDoc.querySelector('#juicy-popover-template').content;
+        const importDoc = (document._currentScript || document.currentScript).ownerDocument;
+        const template = importDoc.querySelector('#juicy-popover-template');
 
-        // Shim Shadow DOM styles if needed
+        let usedShadyCss = false;
         if (window.ShadowDOMPolyfill) {
```

Comment:
It seems to me that `window.ShadowDOMPolyfill` is for SDv1. Can be removed.

Reply:
Do you mean this:
```js
if (window.ShadowDOMPolyfill) {
    WebComponents.ShadowCSS.shimStyling(template.content, 'juicy-popover');
} else if (window.ShadyCSS) {
    usedShadyCss = true;
    ShadyCSS.prepareTemplate(template, 'juicy-popover');
}
```
can be cut down to this:
```
WebComponents.ShadowCSS.shimStyling(template.content, 'juicy-popover');
```

Reply:
After second thought, I think this is what we can leave:

Reply:
If I understand correctly, the shim for Shadow DOM called ShadyCSS does not run implicitly, but it has to be called manually in code.

Reply:
At some point in the past it wasn't run implicitly for vanilla CE and SD, but as I noticed recently in `starcounter-include` it does run. So I'll double check this.